style.css

```diff
@@ -0,0 +1,16 @@
+#nav-bar{
+    position: fixed;
+    top: 0;
+    }
+    
+    #header{
+        display: flex;
+        align-items: center;
+        justify-content: space-around;
+    }
```

Comment:
Please, try to standardize your code for easy reading. As in the first part, I suggest using a regular tab and aligning all items.

Reply:
Hi, @HFG43!! 👋
- Well done on making inline comments with suggestions and reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - This is a polite suggestion: "I think it would be better to use `nav` nested of your `header`". 

- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.

- [ ] ***[OPTIONAL]*** You can add any resource as a link in the inline comment also. It's a better way to give a better resource and suggestions.